blaze/expr/tests/test_reductions.py

```diff
@@ -97,3 +97,33 @@ def test_max_min_on_datetime_and_timedelta(reduc, measure):
 
 def test_reduction_naming_with_generated_leaves():
     assert symbol('_', 'var * float64').sum()._name == 'sum'
+
+
+@pytest.mark.parametrize(
+    ['reduction', 'group_by', 'sort_by', 'preceding', 'following'],
+    list(product(
+        ['sum', 'mean', 'count'],
+        list('ab'),
+        list('ab'),
+        [None, 0, 3],
+        [None, 0, 3]
+    ))
+)
+def test_basic_over(reduction, group_by, sort_by, preceding, following):
+    t = symbol('t', 'var * {a: int64, b: ?float64}')
+    expr = getattr(t.b, reduction)().over(
+        group_by=group_by,
+        sort_by=sort_by,
```

Comment:
ah, I see. This makes sense then. When we change the `groupby` api to be more pandas like, do you forsee changing the over syntax to by based on that object?